src/python/src/grpc/_adapter/_c/module.c

```diff
@@ -0,0 +1,26 @@
+#include <stdlib.h>
+
+#include <grpc/grpc.h>
+
+#include "grpc/_adapter/_c/types.h"
+
+static PyMethodDef c_methods[] = {
```

Comment:
Does this have to be defined or can you just write its value at its one use-site?